CONTRIBUTING.md

```diff
@@ -123,6 +123,7 @@ The exact steps for an exemplary `0.2.0` release might look like this:
   1. Initially, the version on the latest master is `0.2.0-dev`
   2. A new `v0.2.0` branch is created for the release
   3. On master, the version is bumped to `0.3.0-dev`
+     and the `-dev` suffix is removed from the `0.2.0` changelog
```

Comment:
Should it be `0.2.0-dev`?

Reply:
Either way would be fine, but we can clarify, sure. This is more talking about the "changelog of the 0.2.0 version".